integration_tests/dbt_project.yml

```diff
@@ -8,33 +8,33 @@ vars:
 
   ## Use this variable to run the project with synthetic data loaded as seeds
   # otherwise set to false and set the input vars below
-  test_data_override: true
+  test_data_override: false
 
 
   ## Update these vars to use your own data as input, do not comment out
   # enabled logic has been added to the sources config
-  input_database: medicare_lds_five_percent
-  input_schema: input_layer
+  input_database: tuva
+  input_schema: tuva_claims
 
   tuva_last_run: '{{ run_started_at.astimezone(modules.pytz.timezone("UTC")) }}'
   ## All Tuva Data Marts
-  tuva_marts_enabled: true
+  #tuva_marts_enabled: true
 
   
-  ## Individual Tuva Data Marts
-  # acute_inpatient_enabled: false
-  # ccsr_enabled: false
-  # cms_chronic_conditions_enabled: false
-  # claims_date_profiling_enabled: false
-  # cms_hcc_enabled: false
-  # core_enabled: false
-  # data_profiling_enabled: false
-  # member_months_enabled: false
-  # pmpm_enabled: false
-  # quality_measures_reporting_enabled: false
-  # readmissions_enabled: false
-  # service_category_grouper_enabled: false
-  # tuva_chronic_conditions_enabled: false
+  ## The Tuva Project can support claims and clinical data.  
+  # By default, all models are disabled.
+  # Use the vars below to enabled or disable the pieces of The Tuva Project 
+  #    as needed.
+  # clinical_enabled: true
+  # claims_enabled: true
+  # claims_preprocessing_enabled: true
```

Comment:
I think we should add a note that people can optionally turn on/off individual data marts and clarify that the above vars will turn on all data marts specific to that data source.

Reply:
I added some new language here but it doesn't seem to have updated in this view in GitHub.  I'm going to resolve this conversation to see if I can get the new wording to show up.

Reply:
I added some new language here.  Hopefully this helps.